lib/MessageDigest-browser.js

```diff
@@ -3,11 +3,7 @@
  */
 'use strict';
 
-require('setimmediate');
```

Comment:
It's weird that this was put here, as I think it may have been required for some other reason (i.e., this function isn't supported in browsers but we use it elsewhere). @davidlehn, can you speak to this?

Reply:
`setImmediate` is used in the code, so a polyfill is needed.  See `lib/URDNA2015.js`.  This polyfill replaced custom code and was added in:
https://github.com/digitalbazaar/rdf-canonize/commit/4e10528f9c1bc580d243121cb24353b3c9687e61

The browser testing of this package is indirect through jsonld.js, to avoid complexity of duplicating the testing framework.  A side effect of that seems to be confusion as to why this is needed.  jsonld.js tests will fail without this.

I'm not sure why it's in this file in particular.  I probably thought it easier to just use this file rather than create a new empty node file and a browser only file with just that require.

Please restore.  If a comment or moving to a new browser only file would help avoid confusion, that's fine.

Reply:
Ahhh got it, thanks!!

Reply:
@davidlehn interestingly, requiring setimmediate causes to fail a build in my case. I could provide a repro if you are interested and would be interested in understanding why.